vllm/config.py

```diff
@@ -3278,6 +3284,11 @@ def _extract_backend_options(self):
             self.disable_additional_properties = True
 
 
+# For backward compatibility, should remove in v0.10.0
+# given that vllm.config is considered public
+DecodingConfig = StructuredOutputConfig
```

Comment:
Can this also be properly deprecated? Something like

```suggestion
@deprecated("The reason")
class DecodingConfig(StructuredOutputConfig):
    pass
```